spec/game_runner_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'spec_helper.rb'
+require_relative '../lib/game_runner.rb'
+
+RSpec.describe Game, "#run" do
```

Comment:
Where does the `Game` class come from? The file being required is called `game_runner` so the Ruby convention would be to have a class with that same name.

Reply:
✅